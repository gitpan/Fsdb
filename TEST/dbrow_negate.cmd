prog='dbrow'
args='-v "_fullname =~ /John/"'
cmp='diff -cb '
in=TEST/dbrow_ex.in
