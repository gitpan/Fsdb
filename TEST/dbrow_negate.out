#fsdb account passwd uid gid fullname homedir shell
root	*	0	0	Root	/root	/bin/bash
# this is a simple database
#   | dbrow -v _fullname =~ /John/
